File: 3447-clear-digits/clear-digits.py
class Solution:
    def clearDigits(self, s: str) -> str:
        l=[]
        for i in s:
            if ord(i)>=ord("0") and ord(i)<=ord("9"):
                if len(l)!=0:
                    l.pop()
            else:
                l.append(i)
        return "".join(l)

File: 3447-clear-digits/clear-digits.cpp
class Solution {
public:
    string clearDigits(string s) {
        stack<char>st;
        for(int i=0;i<s.size();i++){
            if(s[i]>=97 and s[i]<=122){
                st.push(s[i]);
            }
            else{
                st.pop();
            }
        }
        string res="";
        char ch;
        while(!st.empty()){
            ch=st.top();
            st.pop();
            res=ch+res;
        }
        return res;
    }
};